tests/unit/test_option_parser.py

```diff
@@ -0,0 +1,187 @@
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+
+""" Test the option parsing and filtering form the compilation commands. """
+
+import unittest
+
+from codechecker_lib import option_parser
+from codechecker_lib.option_parser import ActionType
+
+
+class OptionParserTest(unittest.TestCase):
```

Comment:
Would it make sense to always check every attributes of the returned `OptionParserResult` object? Or is this intentional that only a few of them is observed?